_playwright-tests/playwright.config.ts

```diff
@@ -16,12 +16,12 @@ export default defineConfig({
   workers: 1,
   reporter: !!process.env.CI
     ? [
-        ['list'],
```

Comment:
Any reason not to leave the list so when it's running we can watch it click through?

Reply:
I did the same that we have in the frontend. And I don't see it is necessary, you get his "list" view: (as it is also live in currents.dev)
![Screenshot from 2025-02-24 16-13-28](https://github.com/user-attachments/assets/0726f813-88ea-4993-b09f-cbdb70fb4c45)